src/app/products/products-mix/view-product-mix/view-product-mix.component.ts

```diff
@@ -0,0 +1,51 @@
+/** Angular Imports */
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { MatPaginator, MatSort, MatTableDataSource } from '@angular/material';
+
+/**
+ * View product mix component.
+ */
+@Component({
+  selector: 'mifosx-view-product-mix',
+  templateUrl: './view-product-mix.component.html',
+  styleUrls: ['./view-product-mix.component.scss']
+})
+export class ViewProductMixComponent implements OnInit {
+
+  /** Fetched product mix data. */

```

Comment:
```suggestion
  /** Product mix data. */

```

Reply:
Hey @luckyman20, you mean remove it from the component's/files's name?

Reply:
Sorry, I didn't get you. I just want you to change `Fetched product` to `Product`.